website/docs/design-patterns/inline-component-configuration.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Inline Component Configuration Atmos Design Pattern
+sidebar_position: 2
+sidebar_label: Inline Component Configuration
+description: Inline Component Configuration Atmos Design Pattern
+---
+
+# Inline Component Configuration
+
+The **Inline Component Configuration** pattern is used when the [components](/core-concepts/components) in a [stack](/core-concepts/stacks) 
+are configured inline in the stack manifest without [importing](/core-concepts/stacks/imports) and reusing default/base configurations.
```

Comment:
```suggestion
are configured inline in the stack manifest without [importing](/core-concepts/stacks/imports) and reusing default/base configurations. If you're starting with Atmos, this will be the easiest to understand until you need the more advanced features.
```